print-default.properties

```diff
@@ -122,4 +122,13 @@ mosip.print.crypto.p12.password=password@123
 mosip.print.crypto.p12.alias=partner
 
 # verifiable credential
-mosip.print.verify.credentials.flag=true
+mosip.print.verify.credentials.flag=false
```

Comment:
why is this made false ?

Reply:
it's for testing, changing to default value